recipes/anvio-minimal/meta.yaml

```diff
@@ -12,43 +12,38 @@ source:
 
 build:
   number: 0
+  skip: True  # [not py36]
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-  build:
-    - {{ compiler('c') }}
   host:
-    - python >=3,<3.7
-    - pip
+    - python
     - numpy
-    - cython
-    - gsl
+    - setuptools ==41.2.0
+    - wheel ==0.33.6
+    - pip ==19.2.3
   run:
-    - python >=3,<3.7
-    - numpy
+    - python
+    - {{ pin_compatible('numpy') }}
     - bottle ==0.12.13
     - pysam ==0.15.2
```

Comment:
Do the version pinnings really have to be so exact?

Reply:
Many of our dependencies have a similar subset of dependencies like `scipy`, `numpy`, `pandas`, which causes some long conflict chains, especially on `Python 3.7`. We found suggesting users create a new environment with frozen dependencies on `Python 3.6` works best under these circumstances, for now. We are planning to loosen dependencies for the next release and hoping to target `Python 3.7` too. We still have limited experience in distributing on Conda and any suggestions and tips are highly appreciated.